src/pretix/presale/templates/pretixpresale/event/fragment_cart.html

```diff
@@ -212,6 +212,13 @@
                     {% else %}
                         {{ line.price|money:event.currency }}
                     {% endif %}
+                    {% if line.discount and line.line_price_gross != line.price %}
+                        <span class="text-success discounted" data-toggle="tooltip" title="{% trans "The price of this product was reduced because of an automatic discount." %}">
```

Comment:
Perhaps we could give the customer a little bit more details on to why this is discounted... We should be able to create a human readable version of the discount rule?

Reply:
Hmm, this sounds quite tough. Dynamical string building is very hard to do in a way that works across languages. Is this really something users will expect? Do other online shops do this? :thinking:

Reply:
This was something I noticed as well. Is there a way to easily make it possible to enter custom strings when creating the discount rule, comparable to shop descriptions and such, so a shop owner can set their own explanation in their configured languages?
I do think other shops use discounts as incitement, so they would make a grand proclamation somewhere, that it is possible to get discounts. This is not the place to advertise them, so it would also be fine if left as is, I believe.

Reply:
I did not get around to this before the merge now, but I think the custom-set explanation ("public description") is a great solution! We can always implement it later on. Would you maybe interested in doing that, @ser8phin? :)